woocommerce/woocommerce
# Remove broken workflow for bypassing changelog-only PRs

### All Submissions:






### Changes proposed in this Pull Request:

This PR removes the bypass for changelog-only PRs, because rather than bypassing checks in PRs that only contain changelog changes, it bypassed checks in PRs containing changelog changes. The existing bypasses in `.github/workflows/status-check-bypass-automation.yml` should be sufficient for allowing our automations to not require checks.



### How to test the changes in this Pull Request:




1. I've created a test PR to confirm that the issues exists. Look at #37215 and note that `Status Check Bypass for Changelog Only Changes / Runs E2E tests. (pull_request)`, for example, passes, making the E2E tests not required.
2. Fork this repo, delete the `.github/workflows/status-check-bypass-changelog-only.yml` file in your fork, and create a PR similar to #32715 in your fork. Verify that that changelog-only change bypasses never run.



### Other information:




### FOR PR REVIEWER ONLY:
